### ISSUE TITLE
A11y_AzuredataStudio_ManageAccessForFile_ScreenReader:Name property of check box is not announced by Voice Over .

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7943]
GitHubTags: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG4.12;#WCAG4.2.1;#sktopApp;#Win10;#Voice Over;
**Environment Details: **
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit:6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: MacOS Mojave 10.14.6
Additional Details:
Tool Name : VoiceOver
Steps to Reproduce:

  1.Install and Launch the application the azure data studio application.
 2.Connet to the servers with the required credentials.
 3.Navigate to the control panel and click on the “Jupyter books” icon.
 4.Navigate till HDFS folder and right click on it and click on manage access.
 5.Start Voice Over and observe the announcement.
Actual:
Voice Over is Not announcing name property of the check box.
Expected:
Voice Over should announce name property of the check box.
User Impact:
Screen Reader user will not able to relate with the Check box.
Attachment for Reference:
[7943_A11y_AzuredataStudio_ManageAccessForFile_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3761509/7943_A11y_AzuredataStudio_ManageAccessForFile_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by hannahqin** [comment]
Closing as a duplicate of https://github.com/microsoft/azuredatastudio/issues/7799.

**Comment by vinodbk197** [comment]
Verified this issue on:
Azure Data Studio
Version: 1.15.0-insider (user setup)

The issue is no longer to reproduce, Hence closing this issue.
[7943_Closed.pptx](https://github.com/microsoft/azuredatastudio/files/4138879/7943_Closed.pptx)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-May2019;
